packages/pwa-kit-create-app/scripts/build.js

```diff
@@ -8,19 +8,26 @@ const p = require('path')
 const sh = require('shelljs')
 const fs = require('fs')
 const os = require('os')
+const tar = require('tar')
 
 sh.set('-e')
 sh.config.silent = false
 
 const monorepoRoot = p.resolve(__dirname, '..', '..', '..')
-const templatePath = p.resolve(__dirname, '..', 'template')
-const helloWorldTemplatePath = p.resolve(__dirname, '..', 'template-hello-world')
+const templatesDir = p.resolve(__dirname, '..', 'templates')
```

Comment:
was `template` renamed to `templates`? if that's the case, the `.eslintignore` still references `template`

Reply:
Yes it was! Nice catch. Will fix. Strictly speaking, it won't matter any more in any case though – the source is now bundled in a tarball that `eslint` can't/won't inspect anyway.

Reply:
I've actually dropped the file – there's no reason to keep it at all now, since `template` was the only entry.